packages/css-library/src/stylesheets/base/va.scss

```diff
@@ -348,6 +355,7 @@ article > h1 {
   max-width: $lead-max-width;
 
   p, a {
+    font-family: $font-serif;
```

Comment:
`va-introtext p` is used on a bunch of different hub pages. Without explicitly setting font-family to `$font-serif` these paragraphs will render source sans pro instead of bitter which is how they're currently rendered. This is because our uswds-typography stylesheet will override font-family.